code/modules/mob/living/ssd_indicator.dm

```diff
@@ -6,7 +6,7 @@
 		icon_state = "default0"
 		layer = FLY_LAYER
 
-/mob/living/var/obj/effect/decal/ssd_indicator/indicator
+var/static/obj/effect/decal/ssd_indicator/indicator
```

Comment:
this should be a variable tied to mob/living itself, not a global variable unless you plan on using the helpers